Makefile

```diff
@@ -13,7 +14,11 @@ bash:
 
 .PHONY: ci
 ci:
-	@$(BASH_TEST) "bin/rails db:setup db:migrate ci"
+	@$(BASH_TEST) "bin/rails db:setup db:migrate assets:precompile ci"
+
+.PHONY: ci_local
+ci_local:
+	@$(MAKE) "CI=true RAILS_ENV=test ci"
```

Comment:
I'm behind on my `makefile` syntax - is this force setting `CI=true RAILS_ENV=test` before calling `make ci`? I think `CI=true` means that this is the CI run, which for us means Jenkins. I would expect this to _not_ be set when running locally.

Reply:
running just `make ci` locally fails with error messages that are resolved by setting `CI=true` and `RAILS_ENV=test`

I can remove this addition if as it was mostly for ease of use locally

Reply:
What was the original error? Perhaps there's a better solution. As it stands, setting `CI=true` & naming the make target `ci_local` is a bit of a confusing oxymoron - they mean the opposite of one another.

Reply:
Removed `ci_local`